lib/utils.pm

```diff
@@ -327,7 +327,15 @@ sub unlock_if_encrypted {
         assert_screen("encrypted-disk-password-prompt", 200);
         type_password;    # enter PW at boot
         save_screenshot;
-        assert_screen 'encrypted_disk-typed_password' if $args{check_typed_password};
+        if ($args{check_typed_password}) {
+            unless (check_screen "encrypted_disk-typed_password", 30) {
+                record_info("Invalid password", "Not all password characters were typed successfully, retyping");
+                send_key "backspace" for (0 .. 9);
+                type_password;
+                assert_screen "encrypted_disk-typed_password";
+                save_screenshot;
```

Comment:
you don't need this line when `assert_screen` is used before. Wondering why there are not two screenshots in your VR...perhaps the behavior in openQA changes. still redundant, right?

Reply:
done